api/integrations/github/models.py

```diff
@@ -47,3 +48,19 @@ class Meta:
                 name="unique_repository_data",
             )
         ]
+
+    @hook(BEFORE_DELETE)
+    def delete_feature_external_respurces(
+        self,
+    ) -> None:
+        from features.feature_external_resources.models import (
+            FeatureExternalResource,
+        )
+
+        FeatureExternalResource.objects.filter(
+            feature__in=self.project.features.all(),
+            type__in=[

```

Comment:
Do you think we need to add an index on `FeatureExternalResource.type`?

Reply:
Good idea, I added an index